pages/profile.js

```diff
@@ -2,8 +2,6 @@ import React, { Component } from 'react'
 import join from 'url-join'
 import getConfig from 'next/config'
 import Button from '../components/button'
-import Chance from 'chance'
```

Comment:
For some reason the the bundler wasn't finding the chance module and I have no idea why and didn't want to spend much time on it.